sources/osg/FrameBufferObject.js

```diff
@@ -252,6 +252,10 @@ FrameBufferObject.prototype = MACROUTILS.objectInherit( GLObject.prototype, MACR
         if ( status !== 0x8CD5 ) {
             this._reportFrameBufferError( status );
         }
+        if ( gl.checkFramebufferStatus( gl.FRAMEBUFFER ) !== gl.FRAMEBUFFER_COMPLETE ) {
```

Comment:
I think this could be removed, as I recall to read it is a expensive call, but in https://www.khronos.org/opengles/sdk/docs/man/xhtml/glCheckFramebufferStatus.xml says that: 
"It is strongly advised, thought not required, that an application call glCheckFramebufferStatus to see if the framebuffer is complete prior to rendering. This is because some implementations may not support rendering to particular combinations of internal formats. In this case, GL_FRAMEBUFFER_UNSUPPORTED is returned."